Crypto.py

```diff
@@ -140,7 +142,7 @@ def crypto(view, enc_flag, password, data):
       _err = str(_err)
     if _err.find('unknown option') != -1:
       panel(view.window(), 'Error: ' + _err)
-    elif _err.find("WARNING:") != -1:
```

Comment:
This was the real issue that led to the plugin not working for me - the warning output openssl returned had a space between WARNING and the colon, so it wasn't properly ignored.